src/main/java/uk/gov/companieshouse/web/accounts/model/directorsreport/AddOrRemoveDirectors.java

```diff
@@ -6,6 +6,8 @@
 @ValidationModel
 public class AddOrRemoveDirectors {
 
+    private boolean displayWarningBanner;
```

Comment:
Just wondering if this could be displayLtdWarningBanner as it's specifically for Loans to directors? It's a little pedantic point maybe but just a thought should any other warning banners were to be added 😄

Reply:
can look to change this

Reply:
It's a mute point I guess. The naming suggests one banner. The likelihood is it will always be one banner just makes it more specific to LTD .